manim/camera/three_d_camera.py

```diff
@@ -358,11 +354,9 @@ def transform_points_pre_display(
     def add_fixed_orientation_mobjects(
         self, *mobjects, use_static_center_func=False, center_func=None
     ):
-        """This method allows the mobject to have a fixed orientation,
-        even when the camera moves around.
-        E.G If it was passed through this method, facing the camera, it
-        will continue to face the camera even as the camera moves.
-        Highly useful when adding labels to graphs and the like.
+        """Allow the mobject to have a fixed orientation, even when the camera moves around.
+
+        E.G If it was passed through this method, facing the camera, it will continue to face the camera even as the camera moves. Highly useful when adding labels to graphs and the like.
```

Comment:
```suggestion
        E.G If it was passed through this method, facing the camera, it will continue to face the camera even as the camera moves. 
        Highly useful when adding labels to graphs and the like.
```